# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../acroread/desktop-database.diff
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- Adobe/Reader8/Reader/intellinux/bin/xdg-desktop-menu.orig	2008-06-25 18:17:05.000000000 +0200
+++ Adobe/Reader8/Reader/intellinux/bin/xdg-desktop-menu	2008-09-03 10:01:40.000000000 +0200
@@ -628,7 +628,7 @@
         for x in `echo $PATH | sed 's/:/ /g'` /opt/gnome/bin; do
            if [ -x $x/update-desktop-database ] ; then
               DEBUG 1 "Running $x/update-desktop-database"
-              eval '$x/update-desktop-database'$xdg_redirect_output
+              eval echo '$x/update-desktop-database'$xdg_redirect_output
               return
            fi
         done
